duo_client/admin.py

```diff
@@ -2010,13 +2010,28 @@ def get_group_users(self, group_id, limit=100, offset=0):
         limit - The maximum number of records to return. Maximum is 500. (Optional)
         offset - The offset of the first record to return. (Optional)
         """
-        return self.json_api_call(
+        (limit, offset) = self.normalize_paging_args(limit, offset)
+        if limit:
+            return self.json_api_call(
+                'GET',
+                '/admin/v2/groups/' + group_id + '/users',
+                {
+                    'limit': str(limit),
```

Comment:
I don't think we need str conversions here because normalize_paging_args handles that